test/governance/candidacy.users.test.ts

```diff
@@ -0,0 +1,238 @@
+  }
+
+  // Initialize actors
+  candidate = UserFactory.createUser(
+    Users.GovernanceUser,
+    keyring,
+    bootnode
+  ) as GovernanceUser;
+  voter = UserFactory.createUser(
+    Users.GovernanceUser,
+    keyring,
+    bootnode
+  ) as GovernanceUser;
+  bank = new Bank(sudo);
+
+  // Record new test state
+  state.accounts.push(voter);
+
+  // Fund accounts
+  await voter.addMGATokens(bank.sudoUser, accountFundingAmount);
+  await candidate.addMGATokens(bank.sudoUser, stakeAmount);
+
+  // Subscribe to events
+  candidate.node.subscribeToUserBalanceChanges(candidate);
+  voter.node.subscribeToUserBalanceChanges(voter);
+});
+
+afterAll(async () => {
+  // Tests in files are unlikely to impact each other
+  // but not clearing up over multiple test files will break
+  await cleanState();
+
+  // Stops duplicated bootnode subscriptions creating duplicate logs
+  await bootnode.stop();
+});
+
+describe("Governance -> Candidacy -> Users", () => {
+  test("User can run for candidacy", async () => {
+    const candidatesAddress = candidate.keyRingPair.address;
+
+    await candidate.runForCouncil();
+    await waitForNBlocks(2);
+
+    const termZero = getLastBlocksElectionData();
+
+    expect(flattenArray(termZero.Candidates)).toEqual(
+      expect.arrayContaining([candidatesAddress])
+    );
+  });
+
+  test("Candidate can renounce candidacy", async () => {
+    const candidatesAddress = candidate.keyRingPair.address;
+
+    await candidate.runForCouncil();
+    await waitForNBlocks(2);
+
+    await candidate.renounceCandidacy({Candidate: 1});
+
+    await waitForNBlocks(2);
+
+    const termZero = getLastBlocksElectionData();
+
+    expect(flattenArray(termZero.Candidates)).not.toEqual(
+      expect.arrayContaining([candidatesAddress])
+    );
+  });
+
+  test("Council Member can renounce candidacy", async () => {
+    const candidatesAddress = candidate.keyRingPair.address;
+
+    // Gonzalo runs for Council in Term 0
+    // Eddy votes for Gonzalo
+    await candidate.runForCouncil();
+    await voter.vote([candidate], stakeAmount);
+    await waitForNBlocks(2);
+
+    const termZero = getLastBlocksElectionData();
+
+    expect(flattenArray(termZero.Candidates)).toEqual(
+      expect.arrayContaining([candidatesAddress])
+    );
+
+    // Gonzalo wins the election and is council member!
+    // Gonzalo subsequently renounces candidacy in Term 1
+    await waitForNextTerm();
+
+    const termOne = getLastBlocksElectionData();
+    expect(flattenArray(termOne.Members)).toEqual(
+      expect.arrayContaining([candidatesAddress])
+    );
+
+    const candidateStatus = {Member: candidate.keyRingPair.address};
+    await candidate.renounceCandidacy(candidateStatus);
+
+    // Gonzalo is not automatically running for candidacy in Term 2
+    await waitForNextTerm();
+
+    const termTwo = getLastBlocksElectionData();
+
+    expect(flattenArray(termTwo.Candidates)).not.toEqual(
+      expect.arrayContaining([candidatesAddress])
+    );
```

Comment:
Can you check that now is NOT a member? Or is it not working like that?

Reply:
You're still a member till the term has finished, it just stops you re-running for election